packages/graphql-yoga/src/plugins/use-graphiql.spec.ts

```diff
@@ -17,6 +17,24 @@ describe('GraphiQL', () => {
       expect(result).toMatch(/<title>Test GraphiQL<\/title>/)
     })
 
+    it('renders graphiql when passing query params and trailing slash', async () => {
+      const yoga = createYoga({
+        graphiql: () => Promise.resolve({ title: 'Test GraphiQL' }),
+      })
+      const response = await yoga.fetch(
+        'http://localhost:3000/graphql?query=something+awesome',
```

Comment:
Could you add tests for trailing slash as well?

Reply:
closing this PR, not interested anymore